Makefile

```diff
@@ -79,7 +79,7 @@ lint: check-licence eclint-check
 	@$(foreach dir,$(PKGS),golint $(dir) 2>&1 | $(FILTER_LINT) | tee -a lint.log;)
 	@echo "Checking errcheck..."
 	@go get github.com/kisielk/errcheck
-	@errcheck $(PKGS) 2>&1 | $(FILTER_LINT) | tee -a lint.log
+	@errcheck --ignore "WriteString" $(PKGS) 2>&1 | $(FILTER_LINT) | tee -a lint.log
```

Comment:
there are a series of unchecked WriteString calls in thrift_compile.go in master.   am on 1.9  fwiw
```
Checking errcheck...
backend/repository/thrift_compile.go:457:21:	result.WriteString(codeBlocks[i].Code)
backend/repository/thrift_compile.go:458:21:	result.WriteString("\n\n")
backend/repository/thrift_compile.go:524:20:	result.WriteString(fmt.Sprintf("enum %s {\n", e.Name))
backend/repository/thrift_compile.go:527:21:	result.WriteString(fmt.Sprintf("\t%s = %d%s,\n", items[i].Name, items[i].Value,
backend/repository/thrift_compile.go:531:21:	result.WriteString(fmt.Sprintf("\t%s = %d\n%s", items[l-1].Name, items[l-1].Value,
backend/repository/thrift_compile.go:534:20:	result.WriteString("}" + annotationsCode(annotations, "\t", ""))
backend/repository/thrift_compile.go:543:20:	result.WriteString(fmt.Sprintf("%s %s {\n", string(st.Kind), name))
backend/repository/thrift_compile.go:544:20:	result.WriteString(fieldsCode(st.Fields, curFilePath, "\t", true))
backend/repository/thrift_compile.go:545:20:	result.WriteString("}" + annotationsCode(annotations, "\t", ""))
backend/repository/thrift_compile.go:568:21:	result.WriteStrimake: *** [lint] Error 1
```
https://golang.org/pkg/bytes/#Buffer.WriteString

Reply:
We should handle the errors instead of ignoring `WriteString` which is project-wise. A lazy trivial fix would be assigning return values of those `WriteString` calls to `_`, and it's still better than ignore.